community.md

```diff
@@ -79,6 +79,18 @@ community and for maintaining a queue of development work and as the public
 record of work on the project. We use the mailing lists for development discussions,
 where a lengthy discussions is required.
 
+### Meetings
+
+We host online meetings to provide venues for synchronous discussions about the Arrow project. These discussions usually focus on topics of interest to developers who are contributing to Arrow, but we welcome users of Arrow to join. Currently there are two series of regularly scheduled meetings:
```

Comment:
Do we need to be careful about "Arrow" vs "Apache Arrow" throughout this change?

Reply:
Generally, the first mention on a page should use the full name "Apache Arrow" but subsequent mentions can omit the "Apache." So this is fine.

Reply:
Ah, I'm glad to hear that.